### PR TITLE
fix: fix a bug where patch extensions weren't correctly verified

### PR DESCRIPTION
**Pull request by kormide**
🤦

### CODE REVIEW

File: e2e/verify_patches/test.sh [hunk 1]
```diff
@@ -20,6 +20,23 @@ if ! bazel build $BZLMOD_FLAG //...; then
...
+# Should succeed after adding an ignored patch extension
+touch patches/foo.diff
+if ! bazel build $BZLMOD_FLAG //...; then
+  echo "ERROR: expected 'bazel build $BZLMOD_FLAG //...' to pass"
+  exit 1
+fi
+rm patches/foo.diff
+
+# Should fail after adding a patch that isn't in `patches`
+bazel clean --expunge # Need to invalidate the repository cache to see the missing file
```
Comment (kormide) [inline e2e/verify_patches/test.sh:32]: I noticed this when writing the e2e. We won't detect new patch files that are missing until the repo rule invalidates.

Comment (gregmagolan) [reply e2e/verify_patches/test.sh:32]: you can `bazel sync --only npm` instead of `bazel clean --expunge`; faster

**Review by gregmagolan — APPROVED** [review]

File: e2e/verify_patches/test.sh (hunk 1 above)
Comment (kormide) [reply e2e/verify_patches/test.sh:32]: Bazel sync doesn't work under bzlmod.

Comment (gregmagolan) [reply e2e/verify_patches/test.sh:32]: ohh. i see i see.

you could touch any file that is an input to the npm_translate_lock in that case

**Pull request merged**